log4j2.xml

```diff
@@ -138,34 +127,67 @@
             <DefaultRolloverStrategy fileIndex="min" />
         </RollingFile>
 
+        <RollingFile name="CSP_REPORT"
+                     fileName="${sys:labkey.log.home}/csp-report.log"
+                     append="true"
+                     filePattern="${sys:labkey.log.home}/csp-report.log.%i">
+            <!-- p=priority c=category d=datetime t=thread m=message n=newline -->
+            <PatternLayout>
+                <Pattern>%-5p %-24.24c{1} %d{ISO8601} %24.24t : %m%n</Pattern>
+            </PatternLayout>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+            </Policies>
+            <DefaultRolloverStrategy fileIndex="min" />
+        </RollingFile>
+
+        <RollingFile name="INDEXED_DOCUMENTS"
+                     fileName="${sys:labkey.log.home}/labkey-indexed-documents.log"
+                     append="true"
+                     bufferedIO="false"
+                     filePattern="${sys:labkey.log.home}/labkey-indexed-documents.log.%i">
+            <!-- d=datetime m=message n=newline -->
+            <PatternLayout>
+                <Pattern>%d{ISO8601} %m%n</Pattern>
+            </PatternLayout>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="1 MB"/>
+            </Policies>
+            <DefaultRolloverStrategy max="3" fileIndex="min" />
+        </RollingFile>
+
     </Appenders>
 
     <Loggers>
+        <!--
+            Set all packages to INFO level (org.labkey, org.fhcrc, third-party dependencies) by default
+            and send errors to the labkey-errors.log as well.
+        -->
+        <Root level="info">
+            <AppenderRef ref="CONSOLE"/>
+            <AppenderRef ref="LABKEY"/>
+            <AppenderRef ref="ERRORS"/>
+        </Root>
 
         <!-- category for server side script messages -->
-        <Logger name="org.labkey.api.script.ScriptService.Console" level="${env:LOG_LEVEL_API_SETTINGS:-INFO}">
+        <Logger name="org.labkey.api.script.ScriptService.Console" level="info">
             <AppenderRef ref="SessionAppender"/>
         </Logger>
 
-
         <!-- category for Mule messages -->
         <Logger name="org.mule.MuleManager" level="info"/>
 
         <!--
             <Logger name="org.labkey.pipeline.mule.transformers" level="debug"/>
-         -->
+        -->
 
         <!-- category for jasper messages -->
         <Logger name="org.apache.jasper" level="warn"/>
 
         <!-- application classes -->
         <Logger name="org.fhcrc" level="info"/>
```

Comment:
The two lines above (comment and `org.fhcrc` logger) were removed from the product version of this file since they should be covered by the new `Root` logger. There's no harm in keeping these lines; it's just an inconsistency.